tkltest/generate/ui/generate_with_labels_analysis.py

```diff
@@ -49,7 +51,10 @@ def generate_selenium_api_tests(config, crawl_dir):
         crawl_dir (dict): Root crawl directory created by Crawljax for the current test-generation run
     """
     logging.info('Creating Selenium API tests from paths in {}'.format(os.path.join(crawl_dir, _CRAWL_PATHS_FILE)))
-    app_name = config['general']['app_name']
+
+    # app_name = config['general']['app_name']
+    app_name = _CRAWL_PATHS_FILE[12:-5]
```

Comment:
@divtosz can the hard-coded values be replaced with constants?

Reply:
This is also for testing purposes, following the file format crawl_paths_<webappname>.json, just so files don't get overwritten; before the output files were getting overwritten.